packages/epoxy/src/resolvers-mergers/merge-resolvers.ts

```diff
@@ -24,7 +24,7 @@ export function mergeResolvers<TSource, TContext, T extends ResolversDefinition<
     }
   }
   if (resolversFactories.length) {
-    return ((...args) => {
+    return ((...args: any[]) => {
```

Comment:
All of the changes you did in epoxy in this PR, are moved to `graphql-toolkit` already.